### PR TITLE
Support for ssl

### PR DESCRIPTION
**Pull request by SonicDMG**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ WORKDIR /opt/killrvideo-web
 # Copy package.json for dependencies
 COPY package.json /opt/killrvideo-web/
 COPY npm-shrinkwrap.json /opt/killrvideo-web/
-    
+
```
Comment (jeffreyscarpenter) [inline Dockerfile:19]: doesn't look like this file has actual changes?

File: src/server/utils/cassandra.js [hunk 2]
```diff
@@ -44,7 +44,27 @@ export function getCassandraClientAsync(keyspace, dseUsername, dsePassword) {
       } else {
         logger.info('No detected username/password combination was passed in. DSE cluster authentication method was NOT executed.');
       }
-      
+
```
Comment (jeffreyscarpenter) [inline src/server/utils/cassandra.js:47]: See the question I had on the Java implementation about whether we should continue execution if SSL is enabled but there is a problem with the cert file.

**Review by jeffreyscarpenter — COMMENTED** [review]
This pull request doesn't include any changes to docker configuration, like the PRs for other repos did.

**Review by Adron — APPROVED** [review]
LGTM.

**Pull request merged**